src/frontend/js/modules/auth/index.js

```diff
@@ -0,0 +1,17 @@
+import { getSettings, getProvider } from './settings';
+import OIDCAuth from './oidc/index';
+import BasicAuth from './basic/index';
```

Comment:
I wonder why we didn't use this ES6 import syntax in our previous code.

Reply:
probably because Michael preferred to use require for some reason while writing the migration scripts